TypeCobol/Compiler/Diagnostics/TypeCobolChecker.cs

```diff
@@ -631,9 +631,12 @@ public static void OnNode(FunctionDeclaration functionDeclaration)
             var headerNameURI = new URI(header.Name);
             var functions = functionDeclaration.SymbolTable.GetFunction(headerNameURI, functionDeclaration.Profile);
             if (functions.Count > 1)
-                DiagnosticUtils.AddError(functionDeclaration,
+            {
+                Token declareToken = header.ConsumedTokens.First(t => t.TokenType == TokenType.DECLARE);
```

Comment:
Even better, target the token of the name of the function instead of the `DECLARE` keyword. you can retrieve it with `header.FunctionName.NameLiteral.Token`.

Reply:
done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/4b5e1d0af58323248efbd75657d4b554eb7db081